auto_py_to_exe/web/index.html

```diff
@@ -251,5 +255,18 @@ <h2 data-i18n="ui.title.output">Output</h2>
                 <span class="loading-label">Initializing...</span>
             </div>
         </div>
+        
+        <script>
+            // Check if user's default color scheme is dark
+            if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+                const root = document.querySelector("body");
+                const darkThemeButton = document.querySelector(".dark-theme-button");
+                const lightThemeButton = document.querySelector(".light-theme-button");
+
+                darkThemeButton.style.display = "none";
+                lightThemeButton.style.display = "inline";
+                root.setAttribute('data-dark-theme', '');
+            }
+        </script>
```

Comment:
This should be moved to the bottom of `interface.js` in a similar way to `setupLanguageSelection` - could name it `setupTheme`.

In `setupTheme`:

- Set up the on-click for the theme toggle control
- Check the users default color scheme and apply it